codeeve/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
```

Comment:
@mei-li Should this page actually be inside projects? With the models? There is also there a views.py.